### DISCUSSION TITLE
stoicism/todo/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#4] [General]
# stoicism/todo/

https://shanenull.com/stoicism/todo/

**Comment by shane0** [comment]
# stoic challenge

- [form](https://forms.gle/hX4A1pGULWDjpYdN9)

**Reply by shane0** [reply]
sid smith page

**Reply by shane0** [reply]
https://shanenull.com/stoicism/sid/

**Comment by shane0** [comment]
# Write 3 things you would do if you didn't have to worry about money.

WHY NOT?
WHAt woulD
YOU LIKE TO DO?
Don't let another day tick away, try it now.

1. Meditation retreat at Chung Tai in Taiwan
1. Hike bicycle motorcycle and Jeep around the world hitting every major hiking and equestrian trail
1. Build a cabin and live only off what I produce on my own farm

**Comment by shane0** [comment]
![image](https://github.com/shane0/stoicism/assets/1454458/86a79dc2-ea85-4858-b7c6-f879a8435f86)

**Comment by shane0** [comment]
high bridge

https://maps.app.goo.gl/WmDsa9ZStp4Y7zX87?g_st=ic

**Comment by shane0** [comment]
https://youtube.com/shorts/_f1aporgPhY?si=guvpeBJmDBO9CL8R
Atmospheric biomarkers

**Comment by shane0** [comment]
Fold clothes
Folding clothes
https://www.instagram.com/reel/Czhl5voSrE-/?igsh=eW4zcG93ZHQ4a3Zw

**Comment by shane0** [comment]
Wardrobe challenge
Red wing care
https://youtu.be/MwirvDmfVso?si=r66v3Zv5ZateQ-kY

**Comment by shane0** [comment]
# Comedy

Is but cheeks one word or should I spread them

**Comment by shane0** [comment]
# overshoot

By country https://overshoot.footprintnetwork.org/newsroom/country-overshoot-days/

**Comment by shane0** [comment]
# potato chip effect

- primordial confusion
- i need this or that for satisfaction...
- find stoic references
- solution: meditation pure awareness does not change
- addiction: cola is industrial cleansor also cleans taste buds so addicts drink it endlessly tobacco similar 
- eat chips long enough your mouth hurts and you are sick of potato chips
- this applies to most things
- related to addiction
- related to burn out
- related to stress
- related to anything
- workflow/ effect
- meditation support: helps let go of material sources of hapiness and realization that sources of pleasure can become sources of suffering

**Comment by shane0** [comment]
# Kurt Vonnegut

- [ ] create a command line app for this then a web app

## Shapes of Stories: Movie Examples

**Note:** While Vonnegut himself didn't explicitly list specific movie examples in his writings, we can infer these shapes based on common plot structures and Vonnegut's general descriptions.

| Shape | Description | Movie Examples |
|---|---|---|
| **Rags to Riches** | The protagonist's fortune increases throughout the story. | *Cinderella*, *Rocky*, *The Pursuit of Happyness* |
| **Riches to Rags** | The protagonist's fortune decreases throughout the story. | *The Great Gatsby*, *Citizen Kane*, *Requiem for a Dream* |
| **Man in a Hole** | The protagonist's fortune falls, then rises. | *The Shawshank Redemption*, *The Green Mile*, *Forrest Gump* |
| **Icarus** | The protagonist's fortune rises, then falls. | *King Lear*, *Macbeth*, *A Beautiful Mind* |
| **Boy Meets Girl** | A classic romantic comedy shape, often involving a couple overcoming obstacles to be together. | *When Harry Met Sally*, *Pride & Prejudice*, *La La Land* |
| **From Bad to Worse** | A downward spiral where the protagonist's situation continually deteriorates. | *The Road Warrior*, *A Clockwork Orange*, *12 Angry Men* |
| **Which Way is Up?** | A complex shape where the protagonist's fortune fluctuates rapidly and unpredictably. | *Fight Club*, *The Shining*, *The Lord of the Rings* |
| **Creation Story** | A shape often found in mythological narratives, where the world is created from chaos or nothingness. | *Genesis* (the biblical creation story), *Prometheus*, *Ex Machina* |
| **Old Testament** | A shape inspired by the Old Testament of the Bible, often involving a fall from grace followed by redemption. | *The Book of Job*, *The Godfather*, *The Lord of the Rings* |
| **New Testament** | A shape inspired by the New Testament of the Bible, often involving a sacrificial figure who redeems humanity. | *The Passion of the Christ*, *The Matrix*, *Saving Private Ryan* |

Remember that these are just examples, and many stories might fit into multiple categories or have unique variations. Vonnegut's shapes provide a helpful framework for understanding narrative structure, but it's important to consider the individual nuances of each story.

**Reply by shane0** [reply]
```py

import pandas as pd
import io
import matplotlib.pyplot as plt


story_data = """
time,fortune,event
1,10,Poor upbringing
2,20,Part-time job
3,30,Skill development
4,40,Better job
5,50,Promotion
6,60,Business venture success
7,70,Investment gains
8,80,Inheritance
9,90,Successful business
10,100,Wealthy
"""

# Create a pandas DataFrame from the CSV string using io.StringIO
df = pd.read_csv(io.StringIO(story_data))

def identify_shape(df):
    """Identifies the shape of the story based on the data.

    Args:
        df: A pandas DataFrame containing the story data.

    Returns:
        A string representing the identified shape.
    """
    # Implement logic to identify the shape based on the data
    # For example, you could check if the x values are monotonically increasing
    # and the y values are monotonically increasing or decreasing.

    return "Man in a Hole"  # Replace with actual shape identification logic

def plot_story_with_events(df, shape):
    """Plots the story's shape with event annotations.

    Args:
        df: A pandas DataFrame containing the story data.
        shape: The identified shape of the story.
    """
    plt.plot(df['time'], df['fortune'])
    plt.title(f"Story Shape: {shape}")
    plt.xlabel("Time")
    plt.ylabel("Fortune")

    # Add event annotations
    for index, row in df.iterrows():
        plt.annotate(row['event'], (row['time'], row['fortune']),
                     textcoords="offset points", xytext=(0, 10), ha='center')

    plt.show()

def main():
    shape = identify_shape(df)
    plot_story_with_events(df, shape)

if __name__ == "__main__":
    main()
```
